### ISSUE TITLE
bug: Not getting Crasehes in realtime in Firebase Crashlytics dashboard

### ISSUE THREAD

**Issue by ghost** [capacitor-community/firebase-crashlytics#52]
Firebase Version
**8.8.0**

**Platform(s):**
IOS 15.1 

**Current behavior:**
Not getting realtime crashes on release build uploaded on testflight in firebase crashlytics dashboard

**Expected behavior:**
Gets realtime crasehes on firebase crashlytics dashboard 

Enviorment:
"Testflight build"

```

**Comment by robingenz** [comment]
Hi @JatinPreetAxtria,
thank you for your request.
Please fill out the issue template correctly so I get all the information I need.

**Comment by ghost** [comment]
Updated and please let me if you require any further information

**Comment by robingenz** [comment]
Please fill out the issue template completely, even if you think the information is not needed.
Does any message appear in the Firebase Console? If yes, please attach a screenshot.

**Comment by ghost** [comment]
i have only this information can you please specific which information you want because i cannot see any tempelate

**Comment by robingenz** [comment]
You will see the template when you create a new issue: https://github.com/capacitor-community/firebase-crashlytics/issues/new?assignees=&labels=&template=bug_report.md&title=bug%3A+

Missing: Plugin version, Steps to reproduce, Related code, Other information, Capacitor doctor

Important: Does any message appear in the Firebase Console? If yes, please attach a screenshot.

**Comment by ghost** [comment]
no Robin the issue is just we are not getting crash reports in real time that is the only issue in real time

**Comment by BL-marcusrogers** [comment]
I had this same problem. Crashlytics was picking up the crash on the device, but in the log I got "Skipping logging Crashlytics event to Firebase, no Firebase Analytics." No crashes were coming through to Analytics or Crashlytics. I'm using AngularFire in my project and already had Analytics set up and working through that, but it seems that is not enough - I needed to explicitly add Analytics to my Android app. I added the following to the app-level build.gradle dependencies section:

`implementation platform('com.google.firebase:firebase-bom:29.1.0')`
 `implementation 'com.google.firebase:firebase-analytics'`

Then delete your Virtual Device or delete cache and app on your physical device, Invalidate Caches in Android Studio, and do a full build and npx cap sync. This got the crashes to start showing up in Crashlytics for me.
